packages/cozy-device-helper/src/plugins.ts

```diff
@@ -18,10 +18,10 @@ export const hasSafariPlugin = (): Promise<boolean> => {
 }
 
 /**
- * Check if the Cordova's cordova-plugin-network-information plugin is installed
- * @see https://cordova.apache.org/docs/en/latest/reference/cordova-plugin-network-information/
+ * Cordova apps are deprecated, we should not use this plugin anymore
+ * Still, we need to expose this function to avoid breaking changes for now
  * @returns {boolean}
  */
 export const hasNetworkInformationPlugin = (): boolean => {
-  return isCordova() && window.navigator.connection !== undefined
+  return false
```

Comment:
This change is not part of the ESLint upgrade, would be great to inform about it in the commit message, or even better, to split this into another commit